### PR TITLE
feat: navbar & sidebar

### PR DESCRIPTION
**Pull request by himadri2110**
Components Added:
- Navbar
- Sidebar

Pages: 
- Home (dummy)
- Label (dummy)

### CODE REVIEW

**Review by poornima-alugubelly — APPROVED** [review]
Good naming and structure good to merge 👍

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+      <div className="page-wrapper">
+        <Navbar />
+
+        <section className="main-section">
+          <Sidebar />
+
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="/label" element={<Label />} />
+          </Routes>
```
Comment (msdeshmukh009) [inline src/App.js]: You can make a separate component for routes.

**Review by msdeshmukh009 — APPROVED** [review]
The code looks 🔥

**Review by sakshamAK — APPROVED** [review]
You have written a clean code. I liked that you've used barrel exports. Folder structure is good too.  Nicely done!

File: src/App.js (hunk 1 above)
Comment (himadri2110) [reply src/App.js]: Done, thank you!

**Pull request merged**